<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>547</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Render Master</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_scene">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_scene"/>
    <addaction name="actionOpen_scene"/>
    <addaction name="separator"/>
    <addaction name="actionSave_scene"/>
    <addaction name="actionLoad_Scene"/>
    <addaction name="actionScene_Reload"/>
    <addaction name="actionClose_Scene"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Render_Master"/>
   </widget>
   <addaction name="menuNew_scene"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionactionCursorSelect"/>
   <addaction name="actionactionManipulatorTransform"/>
   <addaction name="actionactionManipulatorRotate"/>
   <addaction name="actionManipulatorScale"/>
  </widget>
  <action name="actionNew_scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New Scene...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Scene...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCreateGameObject">
   <property name="text">
    <string>GameObject</string>
   </property>
  </action>
  <action name="actionGame_Viewport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Game Viewport</string>
   </property>
  </action>
  <action name="actionScene_Viewport">
   <property name="text">
    <string>Scene Viewport</string>
   </property>
  </action>
  <action name="actionScene">
   <property name="text">
    <string>Scene</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actionAssets">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>View Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_Render_Master">
   <property name="text">
    <string>About Render Master</string>
   </property>
  </action>
  <action name="actionBuild_Standalone">
   <property name="text">
    <string>Build</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionProject_Settings">
   <property name="text">
    <string>Project Settings</string>
   </property>
  </action>
  <action name="actionScene_edit">
   <property name="text">
    <string>Game Designer</string>
   </property>
  </action>
  <action name="actionShowreel">
   <property name="text">
    <string>Showreel</string>
   </property>
  </action>
  <action name="actionManipulatorScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/manipulator_scale.png</normaloff>:/icons/manipulator_scale.png</iconset>
   </property>
   <property name="text">
    <string>ManipulatorScale</string>
   </property>
  </action>
  <action name="actionactionManipulatorTransform">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/manipulator_transform.png</normaloff>:/icons/manipulator_transform.png</iconset>
   </property>
   <property name="text">
    <string>actionManipulatorTransform</string>
   </property>
  </action>
  <action name="actionactionManipulatorRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/manipulator_rotate.png</normaloff>:/icons/manipulator_rotate.png</iconset>
   </property>
   <property name="text">
    <string>actionManipulatorRotate</string>
   </property>
  </action>
  <action name="actionScene_Reload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scene Reload</string>
   </property>
  </action>
  <action name="actionLoad_Scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_Scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Scene</string>
   </property>
  </action>
  <action name="actionactionCursorSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/icons/cursor_blue.png</normaloff>:/icons/cursor_blue.png</iconset>
   </property>
   <property name="text">
    <string>actionCursorSelect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
